src/Components/ApiRestHandler/requestHandler.js

```diff
@@ -1,6 +1,6 @@
 import axios from 'axios';
 
-const apiURL = process.env.REACT_APP_BRANCH === 'test' ? 'https://backend-fullapirest-test.onrender.com/api/' : 'https://backend-fullapirest.onrender.com/api/';
+const apiURL = process.env.REACT_APP_BRANCH === 'test' ? 'http://localhost:9000/api/' : 'http://localhost:9000/api/';
```

Comment:
Both are the same route 'http://localhost:9000/api/'
And I think here should be the onrender backend server URL